diff --git a/CMakeLists.txt b/CMakeLists.txt
index deb7770..e233eae 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -50,7 +50,8 @@ endif()
 option( BUILD_SHARED_LIBS "Toggle building shared libraries." OFF)
 
 # Include executables as part of the build
-option(NIFTI_BUILD_APPLICATIONS "Build various utility tools" ON)
+option(NIFTI_BUILD_APPLICATIONS "Build various utility tools" OFF)
+set(NIFTI_BUILD_TESTING FALSE)
 mark_as_advanced(NIFTI_BUILD_APPLICATIONS)
 
 #When including nifti as a subpackage, a prefix is often needed to avoid conflicts with sytem installed libraries.
